include/bluetooth/audio.h

```diff
@@ -988,6 +988,17 @@ int bt_audio_unicast_group_create(struct bt_audio_chan *chans,
 				  uint8_t num_chan,
 				  struct bt_audio_unicast_group **unicast_group);
 
+/** @brief Delete audio unicast group.
+ *
+ *  Delete a audio unicast group as a central. All channels in the group shall
+ *  be in an idle or configured state.
```

Comment:
Proposal: "in idle state or configured state".  (I think the "an" should be removed.)

Reply:
Replaced "an" with "the"